addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2614,3 +2614,15 @@ we-button-group.o_grid we-selection-items {
         pointer-events: none;
     }
 }
+
+// OUTDATED SNIPPET ALERT
+we-customizeblock-option.snippet-option-VersionControl {
+    position: fixed !important;
+    top: calc(#{$o-we-sidebar-top-height} + var(--snippets-menu-height));
+    padding: 0 !important;
+    z-index: 2;
```

Comment:
`z-index` is still lower than some `.active` buttons, e.g. when a bold text / link is selected on the snippet:

![master-rev--sobo_b](https://github.com/odoo/odoo/assets/25032876/cef131d9-a717-41ae-bc85-f41033787e70)

Maybe the `$zindex-fixed` value should be used here :thinking:.

Reply:
Ah... I thought I set it high enough to avoid that :open_mouth: I will change that :+1: